int8 duration
geometry_msgs/Pose pose